tests/protocol/test_wire.py

```diff
@@ -316,68 +316,6 @@ def test_get_in_vm_artifacts_profile_default(self, *args):
             self.assertEqual(dict(onHold='true'), in_vm_artifacts_profile.__dict__)
             self.assertTrue(in_vm_artifacts_profile.is_on_hold())
 
-    def test_fetch_manifest_fallback(self, *args):
```

Comment:
Already covered in `test_fetch_manifest_should_use_host_channel_when_direct_channel_fails_and_set_it_to_default`